classes/observer.php

```diff
@@ -33,6 +33,7 @@
 use local_annoto\annoto_completion;
 use local_annoto\annoto_completiondata;
 
+// To see changes on the events need to purge caches.
```

Comment:
please remove the comment, it's not related to this class.
Can add it to our moodle dev REAME instead